osp/wrappers/wet_synthesis_wrappers/cfd_pbe_session.py

```diff
@@ -116,12 +114,6 @@ def _initialize(self, root_cuds_object, buffer):
             os.getcwd(), "simulation-wetSynthCfdPbe-%s" % root_cuds_object.uid)
         shutil.copytree(self._case_template, self._case_dir)
 
-        if self._engine == "fluent":
```

Comment:
So you are removing support for Fluent? Not my business anyway, just curious.

Reply:
Yes